### PR TITLE
Top bar

### PR DESCRIPTION
**Pull request by SaptakS**
@mariobehling please review #1319 

**medium screen resolution**
screenshot from 2016-06-30 07 59 29 copy

**Small screen resolution**
screenshot from 2016-06-30 07 59 51

**Create Event Button**
screenshot from 2016-06-30 07 59 29

### CODE REVIEW

File: .bowerrc [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
-    "directory": "open_event/static/admin/lib"
+    "directory": "open_event/static/admin/lib",
+    "proxy": "http://172.31.1.4:8080",
```
Comment (niranjan94) [inline .bowerrc]: I think the proxy needs to be removed ?

File: open_event/helpers/data_getter.py [hunk 2]
```diff
@@ -178,10 +178,10 @@ def get_sessions_of_user(upcoming_events=True):
         """
         if upcoming_events:
             return Session.query.filter(Session.speakers.any(Speaker.user_id == login.current_user.id)).filter(
-                Event.state != 'Completed')
+                Session.start_time >= datetime.datetime.now())
         else:
             return Session.query.filter(Session.speakers.any(Speaker.user_id == login.current_user.id)).filter(
-                Event.state == 'Completed')
```
Comment (niranjan94) [inline open_event/helpers/data_getter.py]: The previous logic was correct. It is supposed to be sessions in upcoming **events** and past **events**. See the original issue #703 .

File: .bowerrc (hunk 1 above)
Comment (SaptakS) [inline .bowerrc]: My mistake.. sorry..

File: open_event/helpers/data_getter.py (hunk 2 above)
Comment (SaptakS) [inline open_event/helpers/data_getter.py]: But I think upcoming event means the one whose date is after the current date. And here state=completed shows the events which are live. So I think the new logic is correct. @mariobehling what do you think?

Comment (niranjan94) [inline open_event/helpers/data_getter.py]: I meant, the date comparison has to be against event and not session.

Comment (SaptakS) [inline open_event/helpers/data_getter.py]: But if session is in an event which is at a date later to current date, then wouldn't the session will also have a date later to current date?

**Pull request merged**